CMakeLists.txt

```diff
@@ -35,8 +35,8 @@ if(CMAKE_GENERATOR MATCHES Win64*)
   endif()
 endif()
 
+set(CMAKE_DOTNET_TARGET_FRAMEWORK_VERSION "4.5.2")
 set(DOTNET_TARGET_FRAMEWORK_VERSION "4.5.2")
```

Comment:
This is line should be deleted.

Reply:
This is a partial conversion to using `CMAKE_DOTNET_TARGET_FRAMEWORK_VERSION` , there are some remaining csproj.in files belonging to `cli` test framework that will be addressed in https://issues.apache.org/jira/browse/GEODE-5818  until then we need both

Reply:
@pivotal-jbarrett how's this look now?